web-registry/src/lib/queries/react-query/coingecko/simplePrice/simplePriceQuery.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  fetchSimplePrice,
+  GECKO_TOKEN_IDS,
+  GECKO_USD_CURRENCY,
+} from 'lib/coingecko';
+
+import {
+  ReactQuerySimplePriceParams,
+  ReactQuerySimplePriceResponse,
+} from './simplePriceQuery.types';
+
+export const getSimplePriceQuery = ({
+  ...params
+}: ReactQuerySimplePriceParams): ReactQuerySimplePriceResponse => ({
+  queryKey: ['simplePrice', GECKO_TOKEN_IDS],
+  queryFn: async () => {
+    const simplePrice = await fetchSimplePrice({
+      ids: GECKO_TOKEN_IDS,
+      vsCurrencies: GECKO_USD_CURRENCY,
+    });
```

Comment:
It seems that we use this function by default with those parameters, which seems more like our default configuration... maybe it would make sense to export from `lib/coingecko` a wrapper function with the parameters already set (option 1), or even the `fetchSimplePrice` function itself with the parameters optional and a default value (option 2). In this way we could call it directly without parameters like:
```
const simplePrice = await fetchSimplePrice();
or
const simplePrice = await fetchSimplePrice({});
```
* option 1
```
export const fetchSimplePriceDefault =
  async (): Promise<FetchSimplePriceResponse> =>
    await fetchSimplePrice({
      ids: GECKO_TOKEN_IDS,
      vsCurrencies: GECKO_USD_CURRENCY,
    });
```

* option 2
```
type FetchSimplePriceParams = {
  ids?: string;
  vsCurrencies?: string;
};

export const fetchSimplePrice = async ({
  ids = GECKO_TOKEN_IDS,
  vsCurrencies = GECKO_USD_CURRENCY,
}: FetchSimplePriceParams): Promise<FetchSimplePriceResponse> => {
...
```

Reply:
Option 2 might make sense I think, it still allows doing custom calls while simplifying the default case.

Reply:
I've implemented option 2.